ui/js/directives/treeherder/clonejobs.js

```diff
@@ -526,23 +526,31 @@ treeherder.directive('thCloneJobs', [
 
             //Empty the job column before populating it
             jobTdEl.empty();
-
             var jgObj, jobGroup, i;
             for (i=0; i<jobGroups.length; i++) {
 
                 jgObj = jobGroups[i];
-
                 if (jgObj.symbol !== '?') {
                     // Job group detected, add job group symbols
                     jobGroups[i].grkey = jgObj.mapKey;
                     jobGroups[i].collapsed = true;
-                    jobGroup = $(jobGroupInterpolator(jobGroups[i]));
-                    jobTdEl.append(jobGroup);
-
+                    //jobGroup = $(jobGroupInterpolator(jobGroups[i]));
+                    var jobText = "<span class=\"disabled job-group\" title=\"" +
+                              jgObj.name + "\" data-grkey=\"" + jgObj.grkey +
+                              "\"><span class=\"group-symbol\" " +
+                              "ignore-job-clear-on-click=\"\">" + jgObj.symbol +
+                              "</span><span class=\"group-content\"><span class=\"group-job-list\">" +
+                              "<button class=\"btn job-btn btn-xs btn-dkgray key43 filter-shown\" " +
+                              "data-jmkey=\"key43\" ignore-job-clear-on-click=\"\" title=\"" +
+                              jgObj.jobs[0].job_type_name + "\">" + jgObj.jobs[0].job_type_symbol +
+                              "</button> </span><span class=\"group-count-list\">" +
+                              "</span></span></span>";
```

Comment:
So you could have saved some quoting troubles by using the "'" character. :) But a better solution would be to just use the interpolator here. The only difference is that you won't use the jQuery selector to create DOM elements here.

Something like:
```
jobTdHtml += jobGroupInterpolator(jobGroups[i])
```

The only tricky part is that there is some internal content which we need to put inside that object that we're interpolating (we were inserting it into the DOM before via jQuery selectors, below). So you'll need to change `addJobBtnEls` (below) to something like `getJobBtnEls` and make it return a blob of HTML text, which we then insert into the jobGroupInterpolator.

Maybe it would be helpful to describe how interpolators are instantiated here. They're just methods that return a JS string, defined in the clonejobs file here:

https://github.com/mozilla/treeherder/blob/d91cd4ba31d23bc1d4bebae5e040218584bacc72/ui/js/directives/treeherder/clonejobs.js#L46

They are based on templates which we are currently leaving in index.html. For example the job group interpolator is here:

https://github.com/mozilla/treeherder/blob/d91cd4ba31d23bc1d4bebae5e040218584bacc72/ui/index.html#L196